.appveyor.yml

```diff
@@ -5,8 +5,12 @@ environment:
     - TOXENV: "py34"
     - TOXENV: "py35"
     - TOXENV: "py36"
+    - TOXENV: "pypy"
 
 install:
+  # This takes a long time, so it's done only for pypy env.
+  - if "%TOXENV%"=="pypy" choco install python.pypy
```

Comment:
What does this do? Install a newer version of pypy?

Reply:
It installs pypy. It does not come installed on `appveyor` machines. See http://www.appveyor.com/docs/installed-software#python.